packages/web-components/fast-foundation/src/dialog/dialog.template.ts

```diff
@@ -7,31 +7,29 @@ import type { FASTDialog } from "./dialog.js";
  */
 export function dialogTemplate(): ElementViewTemplate<FASTDialog> {
     return html<FASTDialog>`
-        <div class="positioning-region" part="positioning-region">
```

Comment:
This is one I'll need to pull down and play with to put it through it's paces, but moving that to the host here should work for all considerations I can think of.

Reply:
It worked for the ones I considered as well. There's a note in the spec about why it was structured that way, but I couldn't find a way it _didn't_ work like this.